Sources/Core/Subject/Subject.swift

```diff
@@ -325,10 +325,10 @@ class Subject : NSObject {
 
     /// Gets the geolocation dictionary if the required keys are available. Returns nil if not enabled.
     /// - Returns: A dictionary with key-value pairs of the geolocation context.
-    func getGeoLocationDict() -> [String : NSObject]? {
+    public var geoLocationDict: [String : NSObject]? {
```

Comment:
NSObject

Reply:
The same as above, this is for compatibility with Obj-C due to nullable number.s